tests/gordo/client/test_client.py

```diff
@@ -18,6 +18,7 @@
 from mock import patch, call
```

Comment:
Please add TODO comment here for moving this test into `gordo-client` package in the future